flowexperimental/comp/wells/SingleCompWellState_impl.hpp

```diff
@@ -17,6 +17,24 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#ifndef OPM_SINGLE_COMP_WELL_STATE_IMPL_HPP
+#define OPM_SINGLE_COMP_WELL_STATE_IMPL_HPP
+
+// Improve IDE experience
+#ifndef OPM_SINGLE_COMP_WELL_STATE_HPP
+#include <config.h>
+#include <flowexperimental/comp/wells/SingleCompWellState.hpp>
+#endif
+
+#include <opm/input/eclipse/EclipseState/Compositional/CompositionalConfig.hpp>
+#include <opm/input/eclipse/Schedule/Well/Well.hpp>
+
+#include <opm/simulators/utils/BlackoilPhases.hpp>
```

Comment:
I thought part of the purpose of https://github.com/OPM/opm-simulators/pull/6197 was to remove the usage of  `BlackoilPhases.hpp`. Do we still need it?

Reply:
uhm, probably not, i did the changes before I rebased the PR. i'll remedy.